stable/dex/templates/deployment.yaml

```diff
@@ -6,7 +6,11 @@
 {{ $grpcCaBuiltName := printf "%s-ca" $fullname }}
 {{ $grpcCaSecretName := default $grpcCaBuiltName .Values.certs.grpc.secret.caName }}
 
+{{- if .Capabilities.APIVersions.Has "apps/v1" }}
```

Comment:
Why do it this way?
i believe `apps/v1` has existed since 1.10.

Reply:
So it's safe to omit the check and change the apiVersion directly?

Reply:
I would just change the apiVersion to `apps/v1`
We are already using the `apps/` endpoints anyway so this isn't that bit a change.